website/src/controllers/user.js

```diff
@@ -97,7 +97,9 @@ api.score = function(req, res, next) {
       id: id,
```

Comment:
I think we should make this whole section:

``` javascript
task = req.body;
task.notes = task.notes || "This task was created by a third-party service. Feel free to edit, it won't harm the connection to that service. Additionally, multiple services may piggy-back off this task.";
```

That should cover everything. @lefnire is there a reason you limited the parameters that can be passed into the creation part of the score method?